index.html

```diff
@@ -7,6 +7,8 @@
     <link rel="shortcut icon" href="Assets/Favicon.png" type="image/x-icon">
     <title>School Carnival 2K24</title>
     <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.1/css/all.min.css" integrity="sha512-5Hs3dF2AEPkpNAR7UiOHba+lRSJNeM2ECkwxUIxC1Q/FLycGTbNapWXB4tP889k5T5Ju8fs4b1P5z/iB4nMfSQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="Media_q.css">
```

Comment:
Change the file name and link the changed file into the HTML code:

`<link rel="stylesheet" href="responsive.css">`